export.py

```diff
@@ -609,7 +609,7 @@ def run(
     return f  # return list of exported files/dirs
 
 
-def parse_opt():
+def parse_opt(known=False):
```

Comment:
Why did you need to change this?

Reply:
I want the default args of `export.py` and overwrite them with my own arguments. Probably the same reason you changed this in`val.py`.